app/views/content_change_requests/_request_details.html.erb

```diff
@@ -8,54 +8,184 @@
   <span class="form-label">
     <%= f.label :title, "Title of request" %>
   </span>
+  <p class="help-block hint-block--md-6" id="title-hint">
+    Clearly summarise what needs to change - for example, ‘Update phone numbers in Universal Credit guide’ or ‘Create a start page for clean air zones‘.
+  </p>
   <span class="form-wrapper">
-    <%= f.text_field :title, placeholder: "Content change request", class: "input-md-6 form-control" %>
+    <%= f.text_field :title, placeholder: "Content change request", class: "input-md-6 form-control",
+      aria: {
+        describedby: "title-hint"
+      }
+    %>
   </span>
 </div>
 
-<fieldset>
-  <legend>
-    <span>URL affected</span>
-  </legend>
+<div class="form-group">
+  <span class="form-label">
+    <%= f.label :reason_for_request, "What’s the reason for the request?" %>
```

Comment:
Should this be "What's the reason for the change?"? The `reason_for_request` parameter can also be renamed to `reason_for_change` for consistency.

Reply:
Done